contracts/committee/Committee.sol

```diff
@@ -650,7 +668,8 @@ contract Committee is MasterCopyNonUpgradable, ConsensusModule, CommitteeI {
         //            less then the quorum amount.
         if (submissionCount == memberCount ||
             block.number > commitTimeOutBlockHeight) {
-            committeeStatus = CommitteeStatus.RevealPhase;
+            updateCommitteeStatus(CommitteeStatus.RevealPhase);
```

Comment:
tryStartRevealPhase() private functions are called from: `closeCommitPhase()` and `submitSealedCommit()` functions. Both are guarded by `isInCommitPhase` modifier. If this function is called once, it will change state to reveal phase and hence those functions cannot be called anymore. Or I am missing something?

Reply:
Thanks, looks good to me. 👍